src/libs/actions/Policy.ts

```diff
@@ -1562,7 +1562,7 @@ function createWorkspaceFromIOUPayment(iouReport: Report | EmptyObject): string
     const policyID = generatePolicyID();
     const workspaceName = generateDefaultWorkspaceName(sessionEmail);
     const employeeAccountID = iouReport.ownerAccountID;
-    const employeeEmail = iouReport.ownerEmail;
+    const employeeEmail = iouReport.ownerEmail ?? '';
```

Comment:
```suggestion
    const employeeEmail = allPersonalDetails?.[employeeAccountID?.toString() ?? '']?.login;
```
Maybe use this instead.

Reply:
I actually think that we don't need to have an email here, we can just put it empty and it will still work
`employeeEmail = ''`
but I guess it would look weird. But instead of retrieving the email from the personal detail, I just use null-coalescing to keep it simple.